dotcom-rendering/src/lib/assets.ts

```diff
@@ -60,17 +59,16 @@ export const getScriptArrayFromFile = (
 		: legacyManifest[file];
 
 	const scripts = [];
-	const http3Suffix = offerHttp3 ? '?http3' : '';
 
 	if (filename) {
 		scripts.push({
-			src: `${ASSET_ORIGIN}assets/${filename}${http3Suffix}`,
+			src: `${ASSET_ORIGIN}assets/${filename}`,
 			legacy: false,
 		});
 	}
 	if (legacyFilename) {
 		scripts.push({
-			src: `${ASSET_ORIGIN}assets/${legacyFilename}${http3Suffix}`,
```

Comment:
Realistically legacy browsers won't even support HTTP3, so I don't think we really need to add it to the legacy bundles anyway.